beacon-chain/rpc/endpoints.go

```diff
@@ -1266,6 +1268,26 @@ func (s *Service) prysmBeaconEndpoints(
 			handler: server.PublishBlobs,
 			methods: []string{http.MethodPost},
 		},
+		{
+			template: "/prysm/v1/beacon/states/{state_id}/query",
+			name:     namespace + ".QueryBeaconState",
+			middleware: []middleware.Middleware{
+				middleware.AcceptHeaderHandler([]string{api.JsonMediaType, api.OctetStreamMediaType}),
```

Comment:
```suggestion
				middleware.ContentTypeHandler([]string{api.JsonMediaType}),
				middleware.AcceptHeaderHandler([]string{api.OctetStreamMediaType}),
```
The client can send only JSON and we return only SSZ, correct?

Reply:
Ah yes, this is true. Thanks for catching this!